sdk/core/core/inc/az_result.h

```diff
@@ -128,6 +128,8 @@ typedef enum
   AZ_ERROR_HTTP_RESPONSE_OVERFLOW = _az_RESULT_MAKE_ERROR(_az_FACILITY_HTTP, 5),
   AZ_ERROR_HTTP_RESPONSE_COULDNT_RESOLVE_HOST = _az_RESULT_MAKE_ERROR(_az_FACILITY_HTTP, 6),
 
+  AZ_ERROR_HTTP_RESPONSE_CONTAINS_INVALID_HEADERS = _az_RESULT_MAKE_ERROR(_az_FACILITY_HTTP, 7),
```

Comment:
If this is returned when getting a single header, then it should be renamed to something like: AZ_ERROR_HTTP_CORRUPT_RESPONSE_HEADER